datastore/store.go

```diff
@@ -273,7 +273,10 @@ type (
 )
 
 var loadBlobDispatch = map[Hint]loadBlobFunc{
-	ATXDB:       atxs.LoadBlob,
+	ATXDB: func(ctx context.Context, db sql.Executor, key []byte, blob *sql.Blob) error {
+		_, err := atxs.LoadBlob(ctx, db, key, blob)
```

Comment:
We will need blob version handling in fetch client / server, but that can be done later, of course.
Initial `syncv2` version will reuse current blob fetching mechanism

Reply:
Why would it need to care about the version here? The fetched blobs are pushed through the ATX handler which can figure out the version while decoding on its own.